resources/js/processes/screen-builder/screen.vue

```diff
@@ -477,12 +474,18 @@ export default {
       return warnings;
     },
   },
+  created() {
+    this.updateDataPreview = debounce(this.updateDataPreview, 1000);
+  },
   mounted() {
     this.countElements = debounce(this.countElements, 2000);
     this.mountWhenTranslationAvailable();
     this.countElements();
   },
   methods: {
+    updateDataPreview() {
+      this.previewDataStringify = JSON.stringify(this.previewData, null, 2);
+    },
```

Comment:
in order to remove the created on line 409 and to make this method to look nice and have the debounce functionality. I would write it like...
```suggestion
    updateDataPreview: debounce(function () {
      this.previewDataStringify = JSON.stringify(this.previewData, null, 2);
    }, 1000),
```

Reply:
Fixed